<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocTypeEditBase</class>
 <widget class="QWidget" name="DocTypeEditBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>415</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="mTypeListBox"/>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mPbAdd">
         <property name="toolTip">
          <string>Click to add a new document type to the list.</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mPbEdit">
         <property name="toolTip">
          <string>click to edit the selected document type name</string>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mPbRemove">
         <property name="toolTip">
          <string>click to remove the current document type</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Unique Document Number</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Number &amp;cycle:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>mNumberCycleCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mNumberCycleCombo"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="mCounter">
          <property name="text">
           <string>example</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel1_2_2">
          <property name="text">
           <string>Ident template:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel1_2_3_2">
          <property name="text">
           <string>Example ID:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="mExampleId">
          <property name="text">
           <string>example</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="mIdent">
          <property name="text">
           <string>example</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel1_2_3">
          <property name="text">
           <string>Counter:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer3_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mPbEditCycles">
          <property name="text">
           <string>&amp;Edit Number Cyclesâ€¦</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>54</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox2_2">
     <property name="title">
      <string>PDF Creation and Postprocessing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>&amp;Template file:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mTemplateUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mTemplateUrl"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="tmplFileSelectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>W&amp;atermark:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mWatermarkCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mWatermarkCombo">
        <item>
         <property name="text">
          <string>no watermark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>on first page</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>on all pages</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>alternating (3 pages)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>different first and last page (3 pages)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>&amp;Watermark file:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mWatermarkUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mWatermarkUrl"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="watermarkSelectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_4">
        <property name="text">
         <string>Append PDF:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>mWatermarkUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mAppendUrl"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="appendSelectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
